src/morse/updates/api.clj

```diff
@@ -0,0 +1,42 @@
+(ns morse.updates.api
+  "The Telegram Bot API methods for getting updates"
+  (:require [clojure.core.async :as a]
+            [clojure.tools.logging :as log]
+
+            [clj-http.client :as http]
+
+            [morse.api :as m-api]
+            [morse.updates.json :as m-u-json]))
```

Comment:
just `json` is okay

Reply:
Sure. These would be `api` and `updates.json` then. Just stripping off the common prefix.